Sources/StreamChatUI/Resources/en.lproj/Localizable.strings

```diff
@@ -46,6 +46,7 @@
 "message.moderation.delete" = "Delete Message";
 
 "messageList.typingIndicator.typing-unknown" = "Someone is typing";
+"messageList.jump-to-unread-button" = "%d unread";
```

Comment:
sorry for the unsolicited feedback but I think ideally this would be in the `Localizable.stringsdict` file instead to allow for proper localization of pluralization 🙏

Reply:
@nuno-vieira the suggestion makes sense (e.g. if you add "message/s" at the end).

Reply:
yea sorry for not being clearer in my initial comment. its probably ok for English but may be needed for other languages.